gen/wpilibj_frcnetcomm.py

```diff
@@ -74,7 +74,7 @@ def main():
 /**
  * JNI wrapper for library <b>FRC_NetworkCommunication</b><br>.
  */
-@SuppressWarnings(\"MethodName\")
+@SuppressWarnings({\"MethodName\", \"LineLength\"})
 public class FRCNetComm extends JNIWrapper {
```

Comment:
While you are here... should the wrapper class be final with a private ctor too?

Reply:
There are occasional reasons to extend the JNI classes, so no.

Reply:
We could do a protected constructor though. But that would be a separate PR since we would be doing that to all classes.